kolibri/plugins/coach/assets/src/views/reports/ReportsControls.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+
+  <div v-show="$mediaType !== 'print'" class="report-controls">
+    <slot></slot>
+    <div class="report-controls-buttons">
+      <UiIconButton
+        type="flat"
+        :title="coachString('printReportAction')"
```

Comment:
We should implement this tooltip using `KTooltip`, which is more accessible

![image](https://user-images.githubusercontent.com/2367265/69576128-8d5af380-0f80-11ea-9548-df43b310402e.png)